sources/packages/backend/workflow/BPMN/camunda-8/parttime-assessment-decisions.dmn

```diff
@@ -940,34 +940,34 @@
           <text>6</text>
         </inputEntry>
         <outputEntry id="LiteralExpression_1h90dwg">
-          <text>78634</text>
+          <text>86784</text>
         </outputEntry>
         <outputEntry id="LiteralExpression_0lfggsr">
-          <text>0.062400</text>
+          <text>0.043680</text>
```

Comment:
as per the screenshot in the story, there is no `0` in the end, 
![image](https://github.com/bcgov/SIMS/assets/77353155/e08b8055-bfe4-4e2a-b63f-3728d42398a9)

Reply:
That doesn't make any difference in the calculations but I removed them.